partner_restrict_payment_acquirer/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+Open partner form and go to "Sales & Purchases" tab. Add payment acquirers in "Allowed Acquirers" filed.
+
```

Comment:
Please add here:
"When partner opens invoice in portal only allowed acquirers will be available for  payment"